studies/templates/createStudy.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'pages/base.html' %}
+
+{% block title %}{{ _type }}{% endblock %}
+
+{% block header %}
+{% endblock %}
+
+{% block content %}
+<div class="container-fluid">
+<!-- Start Container -->
+  <h2>Suivi d'Étude</h2>
+  Ajouter une étude<hr>
+
+  {% from "_formhelpers.html" import render_field %}
+  <form method="post">
+    <dl>
+        {% for field in formSubmit %}
+        {{ render_field(field, class='form-control') }}
+        {% endfor %}
+    </dl>
+    <input type="submit" name="btn" value="Valider" class='btn btn-warning btn-block'>
+  </form>
+
+  <hr>
+
+  <form method="post">
+  <div class="row">
+    <!-- Start Row -->
+
+    <div class="col">
+      <!-- Start Col -->
+      <h5> Informations sur l'étude</h5>
+        <dl>
+          {% for field in formCreateStudy %}
+          {{ render_field(field, class='form-control') }}
+          {% endfor %}
+        </dl>
+        <!-- End Col -->
+    </div>
+
+    <div class="col">
+      <!-- Start Col -->
+      <h5> Choix du prospect </h5>
+      <dl>
+        {% for field in formProspectChoice %}
+        {{ render_field(field, class='form-control') }}
+        {% endfor %}
+      </dl>
+
+      <h5> Labels </h5>
+      <dl>
+          {% for field in formLabel %}
+          {{ render_field(field, class='form-control') }}
+          {% endfor %}
+      </dl>
+      <!-- End Col -->
+    </div>
+    <!-- End Row -->
+  </div>
+
+  <hr>
+
+  <p><input type="submit" name ="btn" value="Enregistrer" class='btn btn btn-success btn-block'>
+  </form>
+
+  <form method="post">
+    <p><input type="submit" name="btn" value="Annuler" class='btn btn btn-danger btn-block'>
```

Comment:
<p> la balise se ferme où ?

Reply:
C'est pas  un lien plutôt vers la page liste des études au lieu de de faire un submit ?

Reply:
La balise se ferme après
Pour le moment le submit est fonctionnel, à voir ce qui est le mieux

Reply:
should be a link